src/lib/es6.d.ts

```diff
@@ -127,7 +127,7 @@ interface ObjectConstructor {
       * @param target The target object to copy to.
       * @param source The source object from which to copy properties.
       */
-    assign<T, U>(target: T, source: U): T & U;
+    assign<T, U>(target: T, source: U | null | undefined): T & U;

```

Comment:
This (and similar for the overloads below) is needed because `null` and `undefined` are allowed and ignored by Object.assign, but the compiler should not infer `U` as `null` if `null` is passed in, as the result `T & null` is meaningless.

Should I add a comment to that effect?